### DISCUSSION TITLE
Connecting to SOC via Host Name

### DISCUSSION THREAD

**Discussion by lconne81** [Security-Onion-Solutions/securityonion#3374] [Unsupported Versions]
I've just finished setting up Security Onion on VirtualBox and I want to access the Security Onion Console via host name, but putting the host name into my browser's search bar doesn't seem to work for some reason. I know it may have something to do with the IP address tied to my installation, but I'm not sure what setting to change and how to change it.

**Comment by launchdaemon** [comment]
Your machine needs to be able to resolve the hostname of your Security Onion VM.  Either whatever is doing DNS on your network needs to have a record added for the VM or you can add an entry to your `/etc/hosts` file or equivalent depending what OS you are using on your machine.  However it is a VM and if the IP is set via DHCP it may change.  If it is statically set in your VM but still within your DHCP range it may also be assigned to something else which would not be good so you should also ensure there is either an IP reservation for you Security Onion VM or it has an IP set that is not within the DHCP scope.

You will also need to make sure you have allowed your host to access Security Onion with the `so-allow` command in your VM.  Similarly if your machine's IP changes you will need to re-allow the new IP.

**Reply by lconne81** [reply]
During the installation, I set my IP address to be static. So should the IP address I use for my VM and my host name be the same as the IP address for one of my Ethernet adapters for my OS? And what's the best way to successfully change my IP address on Security Onion?

**Reply by launchdaemon** [reply]
When you say your OS, do you mean your host that you are running virtualisation software on?  This will definitely need a different IP address to the one you have assigned Security Onion (a virtualisation guest) or you will have a lot of problems.  There is an experimental utility `so-ip-update` that you can use to change Security Onion's IP if it is a standalone instance.

**Reply by lconne81** [reply]
By OS, I'm referring to my laptop, which I currently have VirtualBox on. The instance that I intend on using is Import. Exactly what IP address should I be using? Does it have to be the same as anything?
